Makefile

```diff
@@ -253,10 +264,13 @@ statusgo-library: statusgo-c-bindings $(LIBWAKU) ##@cross-compile Build status-g
 	@echo "Static library built:"
 	@ls -la build/bin/libstatus.*
 
-statusgo-shared-library: generate
-statusgo-shared-library: statusgo-c-bindings $(LIBWAKU) ##@cross-compile Build status-go as shared library for current platform
+build-libsds: $(LIBSDS)
```

Comment:
Please group all SDS-related targets in a single place, don't drop them around the whole Makefile